mlir/test/Dialect/Arith/int-range-interface.mlir

```diff
@@ -731,18 +731,23 @@ func.func @extui_uses_unsigned(%arg0 : i32) -> i1 {
     func.return %4 : i1
 }
 
-/// Catch a bug that crash in getLoopBoundFromFold when
-/// SparseConstantPropagation loaded in the solver.
-
-// CHECK-LABEL: func.func @caller(%[[ARG0:.*]]
-// CHECK: call @callee(%[[ARG0]])
-// CHECK-LABEL: func.func @callee(%[[ARG0]]
-// CHECK: %[[LOAD:.*]] = affine.load %[[ARG0]]
-// CHECK: scf.for {{.*}} to %[[LOAD]] 
+/// Catch a bug that caused a crash in getLoopBoundFromFold when
+// SparseConstantPropagation is loaded in the solver.
```

Comment:
UltraNIT :tm: : why remove the third slash?

Reply:
#unintentional